deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml

```diff
@@ -38,19 +38,21 @@ spec:
               properties:
                 group:
                   type: string
-                name:
-                  type: string
                 matchLabels:
                   additionalProperties:
                     type: string
                   type: object
                 resource:
                   type: string
+                resourceRef:
+                  type: string
                 version:
                   type: string
               required:
+              - matchLabels
```

Comment:
A PR should contain only relevant changes to that particular task. Please don't include unrelated changes in a PR.
This created a bug -> https://github.com/redhat-developer/service-binding-operator/issues/188